apps/wing/bin/wing

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env -S node --experimental-wasi-unstable-preview1 --no-warnings
```

Comment:
Just for my understanding, it seems like we can simply split these into two scripts. An entrypoint that runs with regular node and the child that's executed with wasm support. Right?

Feels like there is no real additional value to making this a reentrant script.

Reply:
It is technically possible that wing is being run with node that has the flag enabled already. In that case, the logic in this file does a have a use. I also don't think it's worse than separating, but we could do benchmarking to see if there's a perf difference.

Reply:
🤷🏻‍♀️